python/py-resolvelib/Portfile

```diff
@@ -17,16 +17,16 @@ maintainers         nomaintainer
 homepage            https://github.com/sarugaku/resolvelib
 description         ResolveLib at the highest level provides a Resolver class that includes dependency resolution logic.
 
-checksums           rmd160  7447c46371acbc8a9629afecf30f289e2e90de88 \
-                    sha256  40ab05117c3281b1b160105e10075094c5ab118315003c922b77673a365290e1 \
-                    size    17739
+checksums           rmd160  90af8d3b8ae7e387ba6f55942fa0603064f36acd \
+                    sha256  04ce76cbd63fded2078ce224785da6ecd42b9564b1390793f64ddecbe997b309 \
+                    size    21065
 
 long_description \
     ResolveLib at the highest level provides a Resolver class that includes dependency resolution logic. \
     You give it some things, and a little information on how it should interact with them, and it will \
     spit out a resolution result.
 
-python.versions     37 38 39 310 311
+python.versions     37 38 39 310 311 312
```

Comment:
I am going to commit, but it would be useful if you submitted another PR some time soon removing the obsolete 3.7 Python version if that is possible.

Reply:
done - https://github.com/macports/macports-ports/pull/22550